.github/workflows/integration.yml

```diff
@@ -0,0 +1,46 @@
+name: Integration
+on: push
+env:
+  MIX_ENV: test
+  CLDR_BRANCH: ${{ github.ref_name }}
+permissions:
+  contents: read
+jobs:
+  test:
+    services:
+      db:
+        image: postgres:12
+        ports: ['5432:5432']
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        app: ["cldr_calendars", "cldr_calendars_composite", "cldr_calendars_coptic",
+         "cldr_calendars_ethiopic", "cldr_calendars_format",
+         "cldr_calendars_japanese", "cldr_calendars_lunisolar",
+         "cldr_calendars_persian", "cldr_collation", "cldr_currencies",
+         "cldr_dates_times", "cldr_html", "cldr_languages", "cldr_lists",
+         "cldr_locale_display", "cldr_messages", "cldr_numbers",
+         "cldr_person_names", "cldr_plugs", "cldr_print", "cldr_routes",
+         "cldr_sql", "cldr_strftime", "cldr_territories", "cldr_text",
+         "cldr_trans", "cldr_units", "cldr_units_sql", "cldr_utils"]
+        elixir: ["main"]
+        otp: ["maint"]
+    runs-on: ubuntu-latest
+    name: ${{matrix.app}}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: elixir-cldr/${{matrix.app}}
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{matrix.otp}}
+          elixir-version: ${{matrix.elixir}}
+      - run: mix deps.get && mix deps.compile && mix compile --warnings-as-errors && mix test
```

Comment:
I think we should add a dialyzer step here too? I get more issues raised about dialyzer errors that actual code errors! :-)